src/GraphQL.MicrosoftDI/MicrosoftDIGraphQLBuilderExtensions.cs

```diff
@@ -88,4 +88,14 @@ public static IGraphQLBuilder AddScopedSubscriptionExecutionStrategy(this IGraph
             GraphQLParser.AST.OperationType.Subscription);
         return builder;
     }
+
+    /// <summary>
+    /// Verifies that all injected services can be created during GraphQL field execution.
+    /// Requires support for IServiceProviderIsService by the dependency injection framework.
+    /// </summary>
+    public static IGraphQLBuilder ValidateServices(this IGraphQLBuilder builder)
```

Comment:
Maybe add a boolean argument to enable/disable the validation. So the user would be able to enable it in dev or test environment but disable in prod to not hurt the startup time.
```c#
services.AddGraphQL(builder => builder
        .AddSchema<VerifyServicesTestSchema>()
        .ValidateServices(IHostEnvironment.IsDevelopment()));
```

Reply:
Good idea.  Done.